assets/js/modules/analytics-4/datastore/__fixtures__/container-destinations.json

```diff
@@ -0,0 +1,16 @@
+{
+  "6065482709": {
```

Comment:
Is this real user data here? If so, I'd suggest sanitizing it...

Reply:
Updated.